com.unity.netcode.gameobjects/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -141,76 +400,150 @@ public SceneSwitchProgress SwitchScene(string sceneName, LoadSceneMode loadScene
                 throw new Exception($"{nameof(NetworkConfig.EnableSceneManagement)} flag is not enabled in the {nameof(NetworkManager)}'s {nameof(NetworkConfig)}. Please set {nameof(NetworkConfig.EnableSceneManagement)} flag to true before calling this method.");
```

Comment:
This is now a private function so we may want to reword the exception being thrown here, as the user will be calling `[Un]loadScene()` to have hit this.